packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -217,7 +244,7 @@ class DriveCommand extends RunCommandBase {
       }
       throwToolExit('CAUGHT EXCEPTION: $error\n$stackTrace');
```

Comment:
tool exits should give user-actionable information.

Reply:
Switched to `throw`.